targets/esp8266/Makefile.esp8266

```diff
@@ -46,61 +38,35 @@ ifneq ($(MFORCE32),)
 	ESP_CFLAGS += -mforce-l32
```

Comment:
why not have this `MFORCE32`-dependent `ESP_CFLAGS` extension together with the extension above (where `JERRY_CONST_DATA` is defined)